drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -474,6 +474,25 @@ virtio_vdpa_dev_notifier_work(void *arg)
 	return ret;
 }
 
+static void virtio_vdpa_doorbell_relay_disable(struct virtio_vdpa_priv *priv)
```

Comment:
DPDK coding style, should be:
static void
virtio_vdpa...

Reply:
done